Makefile

```diff
@@ -115,8 +115,11 @@ build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 	go build -o bin/csv-merger cmd/csv-merger/csv-merger.go
 
+generate-cert:
+	/bin/bash hack/create_self_signed_cert.sh
+
 .PHONY: run
-run: manifests generate fmt vet ## Run a controller from your host.
+run: manifests generate fmt vet generate-cert ## Run a controller from your host.
```

Comment:
could we just leave generate-cert as a standalone target and not wire it into the 'manifests' target. Reason I ask is this may break the self-hosted Renovate job (which runs a 'make manifests' when it updates the PRs). I don't think we need/want to generate certs when creating the Renovate PRs basically.

Reply:
AH, nevermind. I misread this. It is the 'run' task this is wired into. That is probably fine I think.